pallets/subtensor/src/lib.rs

```diff
@@ -1105,6 +1108,19 @@ pub mod pallet {
 			Self::do_registration(origin, netuid, block_number, nonce, work, hotkey, coldkey)
 		}
 
+		/// Associates the hotkey to this coldkey.
+		/// TODO(const): this call should be costly.
+		#[pallet::call_index(6)]
```

Comment:
- we should not have two calls with same index. we should always add calls at the end of existing calls and increment index.
- also, how are we paying for this call?

Reply:
We'll use an inclusion fee instead of arbitrarily increasing call weight (which will affect block production.)